### PR TITLE
2019-09-03 sriram

### PR DESCRIPTION
**Pull request by srirampadhy8488**
i have changed some code as per your request please review it.

### CODE REVIEW

File: src/main/java/com/matraex/BBB/Android/BBBConsumerAndroid/IntroScreen/IntroScreeen.java [hunk 1]
```diff
@@ -9,12 +9,19 @@ public static void gpsSearch () {
...
+	  WebDriverWait wait= new WebDriverWait(driver, 30);
+	  wait.until(ExpectedConditions.visibilityOfElementLocated((By) element));
+	  element.click();*/
+	  //driver.findElement(By.xpath("//android.widget.Button[contains(@resource-id, 'button2') and @index='0']")).click();// android.widget.Button[contains(@resource-id, 'button1') and @index='1']
+	  //driver.findElement(By.xpath("//android.widget.Button[@text='NO']")).click();
+	  //driver.findElement(By.id("android:id/button2")).click();
+	  //driver.findElement(By.className("//android.widget.Button")).click();
+	  String actval=driver.findElement(By.id("android:id/message")).getText();
+	  System.out.println(actval);
+	  String expval="GPS is disabled in your device. Would you like to enable it?";
```
Comment (sriram8488) [inline src/main/java/com/matraex/BBB/Android/BBBConsumerAndroid/IntroScreen/IntroScreeen.java:22]: we dont need this line here

File: src/main/java/com/matraex/BBB/Android/BBBConsumerAndroid/IntroScreen/IntroScreeen.java [hunk 2]
```diff
@@ -9,12 +9,19 @@ public static void gpsSearch () {
...
- */
-  }
-  public void ram() {
+	  /*WebElement element= driver.findElement(By.xpath("//android.widget.Button[contains(@resource-id, 'button2') and @index='0']"));
+	  WebDriverWait wait= new WebDriverWait(driver, 30);
+	  wait.until(ExpectedConditions.visibilityOfElementLocated((By) element));
+	  element.click();*/
+	  //driver.findElement(By.xpath("//android.widget.Button[contains(@resource-id, 'button2') and @index='0']")).click();// android.widget.Button[contains(@resource-id, 'button1') and @index='1']
+	  //driver.findElement(By.xpath("//android.widget.Button[@text='NO']")).click();
+	  //driver.findElement(By.id("android:id/button2")).click();
```
Comment (sriram8488) [inline src/main/java/com/matraex/BBB/Android/BBBConsumerAndroid/IntroScreen/IntroScreeen.java:18]: we dont need this here

File: src/main/java/com/matraex/BBB/Android/BBBConsumerAndroid/IntroScreen/IntroScreeen.java (hunk 1 above)
Comment (srirampadhy8488) [reply src/main/java/com/matraex/BBB/Android/BBBConsumerAndroid/IntroScreen/IntroScreeen.java:22]: ok thank you i will change this

File: src/main/java/com/matraex/BBB/Android/BBBConsumerAndroid/IntroScreen/IntroScreeen.java (hunk 2 above)
Comment (srirampadhy8488) [reply src/main/java/com/matraex/BBB/Android/BBBConsumerAndroid/IntroScreen/IntroScreeen.java:18]: ok thank you sir

**Pull request merged**